sdk/communication/communication-identity/README.md

```diff
@@ -90,6 +90,23 @@ const client = new CommunicationIdentityClient(endpoint, credential);
 const user = await client.createUser();
 ```
 
+### Creating a new user with customId and get user
+
+Use the `createUser` method to create a new user with customId. This customId can be used to map your application's user identities with Azure Communication Services identities. If you call the CreateUser method again with the same customId, it will return the same user.Id. Therefore, you do not need to store this mapping yourself.
```

Comment:
nit: 
```suggestion
Use the `createUser` method to create a new user with `customId`. This `customId` can be used to map your application's user identities with Azure Communication Services identities. If you call the `CreateUser` method again with the same `customId`, it will return the same `user.Id`. Therefore, you do not need to store this mapping yourself.
```